tgui/src/interfaces/computer_fabricator.ract

```diff
@@ -29,16 +29,7 @@
 			<td><ui-button action='hw_netcard' params='{"netcard" : "0"}' state='{{data.hw_netcard == 0 ? "selected" : null}}'>None</ui-button>
 			<td><ui-button action='hw_netcard' params='{"netcard" : "1"}' state='{{data.hw_netcard == 1 ? "selected" : null}}'>Standard</ui-button>
 			<td><ui-button action='hw_netcard' params='{"netcard" : "2"}' state='{{data.hw_netcard == 2 ? "selected" : null}}'>Advanced</ui-button>
-		{{#if data.devtype != 2}} <!-- No tablets -->
```

Comment:
why remove this comment? was this style of comment not functional? if so use ```{{! No tablets}}``` instead